### ISSUE TITLE
Processing Ore Dictionary not functioning

### ISSUE THREAD

**Issue by UnstableCodeScarlet** [raoulvdberge/refinedstorage#1984]
#### Issue description:
TLDR; Processing Ore Dictionary not fuctioning , Possibly related to #1954 ?
#### What happens:
Does not use the ore dict red dye in processing
#### What you expected to happen:
It to stick the red dye in the chest
#### Steps to reproduce:
1. Make ore dictionary procesing pattern, mine is for Empowered Redstonia with the recipe set for red artificial dye (It was on that when I clicked the plus on jei)
2. Put crafter next to chest with pattern
3. Request, observe lack of using ore dictionaried red dye
...

#### Version (make sure you are on the latest version before reporting):

- Minecraft: 1.12.2
- Forge: 14.23.4.2759
- Refined Storage: 1.6.3

Does this issue occur on a server? [yes/no]
Yes
#### If a (crash)log is relevant for this issue, link it here:

[pastebin/gist/etc link here]
